upi/openstack/include-ipv4.yaml

```diff
@@ -0,0 +1,76 @@
+- name: 'Create the primary cluster network'
```

Comment:
We could run all these things earlier, as part of the very first playbook we run.
What would be remaining would be things like tagging resources, and perhaps attaching FIPs (which could also probably be done earlier).

Reply:
attaching FIPs can't be done earlier because when creating the manifests the installer would complain of addresses already in use

Reply:
@mandre I just updated the PR to make the networks playbook generic and run it before the generation of the manifests regardless if it's a single stack or dual-stack cluster. Let me know what you think about it.
Note that with the previous version, we have been trying to maintain the behavior of single stack IPv4 untouched to reduce the amount of changes on already supported scenarios.

@gryf Let us know what you think as well.

Note that I can easily revert to the previous version as I have it saved locally.

Reply:
That's fine by me. Although I'm wondering if we should just let the neuron to create VIP addresses in case of single stack, or we should support the old way in case of single stack - i.e. letting user provide its own ingress/api VIP addresses in inventory.
If we don't care about old scenario, changes need to be done would be easy, and all the burden regarding vip creation would purely be on playbook side.